src/_cffi_src/openssl/evp.py

```diff
@@ -293,4 +294,10 @@
 #else
 static const long Cryptography_HAS_EVP_PKEY_DH = 1;
 #endif
+
+// OpenSSL 1.1.0 doesn't define this value. But its presence isn't
```

Comment:
Please use the OPENSSL_LESS_THAN_111 symbol in the comment so we can find this with grep and remove it